app/views/reports/index.html.erb

```diff
@@ -15,13 +15,13 @@
   </div>
 
   <div class="form-group">
-    <%= f.label :completed %>
-    <%= f.select(:complete_eq, [['Yes', true], ['No', false]], {}, { :class => 'form-control' }) %>
+    <%= f.label "Limit to complete records (returned to origin)" %>
+    <%= f.check_box :complete_eq, {}, true, false %>
   </div>
 
   <div class="form-group">
-    <%= f.label :has_in_house_repairs %>
-    <%= f.select(:in_house_repair_records_id_null, [['Yes', false], ['No', true]], {}, { :class => 'form-control' }) %>
```

Comment:
@hortongn :  Is this checked (false) v unchecked(true) set properly?  I am slightly confused bu the previous select statement.  Every time I test the report form with the seed data, I get no results.  Can someone else verify that this form works properly?

Reply:
Rishi had this correct.  I was overthinking the not null and got confused by previous Yes/No.